From: akallabeth <akallabeth@posteo.net>
Date: Thu, 7 Apr 2022 09:50:04 +0200
Subject: Fix #7785: Missed GatewayHttpUseWebsockets initialization

In backport #6877 this was missed.

(cherry picked from commit 9ab0667b5470cd16227a2212b57bfc7a71a31043)
---
 libfreerdp/core/settings.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libfreerdp/core/settings.c b/libfreerdp/core/settings.c
index f4b5878..6a7b990 100644
--- a/libfreerdp/core/settings.c
+++ b/libfreerdp/core/settings.c
@@ -488,6 +488,7 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 	settings->GatewayRpcTransport = TRUE;
 	settings->GatewayHttpTransport = TRUE;
 	settings->GatewayUdpTransport = TRUE;
+	settings->GatewayHttpUseWebsockets = TRUE;
 	settings->FastPathInput = TRUE;
 	settings->FastPathOutput = TRUE;
 	settings->LongCredentialsSupported = TRUE;
